.travis.yml

```diff
@@ -5,22 +5,27 @@ rvm:
   - 2.1.10
   - 2.2.6
   - 2.3.3
+  - 2.4.0
```

Comment:
I don't want to keep asking you to keep going back and making changes to something that should be quite simple... But, it looks like the latest is 2.4.1 :p